Engine/source/gui/controls/guiTreeViewCtrl.cpp

```diff
@@ -4952,95 +4989,135 @@ DefineConsoleMethod( GuiTreeViewCtrl, setItemImages, void, ( S32 id, S8 normalIm
    item->setExpandedImage(expandedImage);
 }
 
-DefineConsoleMethod( GuiTreeViewCtrl, isParentItem, bool, ( S32 id ), , "( int id ) - Returns true if the given item contains child items." )
+DefineEngineMethod( GuiTreeViewCtrl, isParentItem, bool, ( S32 itemId ), ,
+   "Returns true if the given item contains child items.\n\n"
+   "@param itemId TreeItemID to check for children.\n"
+   "@return True if the given item contains child items, false if not.")
 {
-   if( !id && object->getItemCount() )
+   if( !itemId && object->getItemCount() )
       return true;
    
-   GuiTreeViewCtrl::Item* item = object->getItem( id );
+   GuiTreeViewCtrl::Item* item = object->getItem( itemId );
    if( !item )
    {
-      Con::errorf( "GuiTreeViewCtrl::isParentItem - invalid item id '%i'", id );
+      Con::errorf( "GuiTreeViewCtrl::isParentItem - invalid item id '%i'", itemId );
       return false;
    }
    
    return item->isParent();
 }
 
-DefineConsoleMethod(GuiTreeViewCtrl, getItemText, const char *, (S32 index), , "(TreeItemId item)")
+DefineEngineMethod( GuiTreeViewCtrl, getItemText, const char *, ( S32 itemId ), ,
+   "Gets the text for a given item.\n\n"
+   "@param itemId TreeItemID to get text of.\n"
+   "@return Text for a given item.")
 {
-   return object->getItemText(index);
+	return(object->getItemText(itemId));
```

Comment:
Yeah where I work we settled on tabs instead of spaces, so making any merges with GG stuff is a pain. Luckily VS has the format feature. I do apologizes though for them!